v1/validator.go

```diff
@@ -29,14 +29,24 @@ func GetSubMux() *goji.Mux {
 	v1.HandleFunc(pat.Get("/"), info)
 	v1.HandleFunc(pat.Post("/validate"), validate)
 
+	v1.HandleFunc(pat.Get("/validate"), func(writer http.ResponseWriter, _ *http.Request) {
+		writer.WriteHeader(405)
+	})
+	v1.HandleFunc(pat.Get("/validate/"), forwardToValidate)
+	v1.HandleFunc(pat.Post("/validate/"), forwardToValidate)
```

Comment:
Shouldn't we offer the main logic at `/validate/` and redirect `/validate` to `/validate/` in order to preserve the previous functionality? It *shouldn't* break robust client code, but many HTTP libraries don't automatically follow redirects.

Reply:
done